### PR TITLE
pokemon_crystal.yml - Fix minor typos

### PR DESCRIPTION
**Pull request by laf2099**

### CODE REVIEW

**Comment by flamesage** [comment]
Thanks for submitting this pull request! Unfortunately our yml mappers are being upgraded to the new `xml` mappers format. So we'd like to keep the yml mappers the way they are. If you'd like to discuss further on Discord about your changes -- feel free!

**Comment by laf2099** [comment]
@flamesage Sorry about that! What is the discord associated with this project? I don't think I am a member yet...

**Comment by flamesage** [comment]
No worries at all! Our discord server can be found on our website at https://gamehook.io

**Review by flamesage — APPROVED** [review]

File: GB/pokemon_crystal.yml [hunk 1]
```diff
@@ -898,7 +898,7 @@ glossary:
     0x06: "Pay Day"
     0x07: "Fire Punch"
     0x08: "Ice Punch"
-    0x09: "ThunderPunch"
+    0x09: "Thunder Punch"
```
Comment (laf2099) [inline GB/pokemon_crystal.yml:901]: @flamesage My change is incorrect according to Scott. "Thunder Punch" should be changed "Thunderpunch" on lines 901 and 921 I believe. I checked Serebii, but not 100% sure if it's correct. Sorry for the mistake

Link: https://www.serebii.net/crystal/tmhm.shtml

**Pull request merged**